common/src/main/java/com/github/khanshoaib3/minecraft_access/mixin/MinecraftClientMixin.java

```diff
@@ -0,0 +1,26 @@
+package com.github.khanshoaib3.minecraft_access.mixin;
+
+import com.github.khanshoaib3.minecraft_access.features.access_menu.AccessMenu;
+import com.github.khanshoaib3.minecraft_access.utils.system.KeyUtils;
+import net.minecraft.client.MinecraftClient;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(MinecraftClient.class)
+public class MinecraftClientMixin {
+    /**
+     * {@link AccessMenu} allows menu functions to be triggered when
+     * no screen opened and alt key with number key are pressed.
+     * We need to suppress original hotbar slot selecting feature.
+     */
+    @Inject(at = @At(value = "INVOKE", target = "Lnet/minecraft/client/network/ClientPlayerEntity;isSpectator()Z"),
+            method = "handleInputEvents",
+            cancellable = true)
+    private void suppressHotbarSlotSelectingWhenAccessMenuFunctionsWork(CallbackInfo ci) {
```

Comment:
This method name is way too long, consider naming it `handleInputEvents` to match where it is being injected into.
```suggestion
    private void handleInputEvents(CallbackInfo ci) {
```